opensrp-anc/src/main/java/org/smartregister/anc/library/listener/ContactTaskDisplayClickListener.java

```diff
@@ -0,0 +1,227 @@
+package org.smartregister.anc.library.listener;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.view.View;
+
+import com.vijay.jsonwizard.constants.JsonFormConstants;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.smartregister.anc.library.R;
+import org.smartregister.anc.library.fragment.ProfileTasksFragment;
+import org.smartregister.anc.library.model.Task;
+import org.smartregister.anc.library.util.ConstantsUtils;
+import org.smartregister.anc.library.util.ContactJsonFormUtils;
+import org.smartregister.util.FormUtils;
+
+import java.util.Calendar;
+
+import timber.log.Timber;
+
+public class ContactTaskDisplayClickListener implements View.OnClickListener {
+    private ProfileTasksFragment profileTasksFragment;
+    private FormUtils formUtils;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'formUtils' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)